### DISCUSSION TITLE
显示被封禁了，理由是重复注册。大佬帮忙看看

### DISCUSSION THREAD

**Discussion by molzy123** [BurnHair/docs#116] [技术支持]
![image](https://github.com/BurnHair/docs/assets/51402021/e306ac2a-a0be-4808-91a4-1b991e8dfd1c)

并没有重复注册，使用的是Github账号注册，什么操作会导致触发重复注册？
我就每天签到一下，都没怎么用就被封了。大佬帮忙看看，0.0

**Comment by BennyThink** [comment]
你发帖的用户 molzy123 ID是9559

你截图里的7626 是 zhenyulin_EADev

所以是重复注册。

**Reply by molzy123** [reply]
被封禁的账号无法发帖，我是被封禁之后再使用另一个账号注册的，（可以看一下注册时间的）这里的封禁重复依据是什么，IP地址吗？

**Reply by BennyThink** [reply]
🤔为啥无法发帖呀，这个是 GitHub discussion，应该和我无关的。

封禁的依据主要就是IP、请求上下文、浏览器特征。不过你现在已经有了 molzy123 我觉得你可以用这个 避免以后发生类似的问题。

**Reply by molzy123** [reply]
因为当初是在公司注册的没注意使用了公司的github账号，我也打算使用自己的个人账号，可是我原本账号的金额怎么办。
是想防止注册机使用github账号注册吗，可是都使用了github账号关联了，这个步骤是不是多余的，因为批量注册github账号本身就不容易吧，github官方应该也会做类似的防护。
并且我也没有类似的行为，能帮忙看看是否是误判了吗。谢谢
或者能有办法帮我把金额转移过去我也可以接受的，麻烦了

**Reply by BennyThink** [reply]
🤣 那你就用 molzy123 吧。如果你的额度是付费的那么可以转移。

**Reply by molzy123** [reply]
😅行吧，目前我用的很少。没有付费，之前那些都是每天签到做活动得来的，虽然没有付费，但是被这样莫名其妙的封禁，反馈了也没办法解封。
没有付费没有售后能理解，但也确实很难建立信任感。新建的这个账号也仅仅只是为了反馈这个问题罢了。
最后感谢你的回答

**Comment by gggwvg** [comment]
我也遇到了同样的问题，我朋友也遇到了。无缘无故就显示被封禁了
